### DISCUSSION TITLE
[8.0.0-beta.3] Best way to invalidate cache?

### DISCUSSION THREAD

**Discussion by dphuang2** [TanStack/table#3993] [Q&A]
Hi,

I am controlling the columns for a Table and running into an issue where I update the `aggregationFn` on a `ColumnDef` but aggregated cell values aren't updated because of a `_groupingValuesCache`

Is there a standard way of dealing with grouping values cache?

**Comment by tannerlinsley** [comment]
Code sandbox would go a long ways here

**Reply by dphuang2** [reply]
sorry 😅 will roll one up when I have time!
